app/controllers/manage_assessments/coverages_controller.rb

```diff
@@ -45,8 +46,12 @@ def coverage_params
         permit(
           :subject_id,
           attachments_attributes: [:id, :file, :_destroy],
-          outcome_coverages_attributes: [:id, :outcome_id, :_destroy]
+          outcome_coverages_attributes: [:outcome_id, :_destroy]
         )
     end
+
+    def update_coverage_params
+      coverage_params.except(:subject_id)
```

Comment:
When I mentored with Paul, he mentioned not liking the `except` keyword because if an attribute is added to `coverage_params` it has to be explicitly excluded from `update_coverage_params`, which is easy to forgot. Although not nearly as clear, is it better to go with the following? 
```
def update_coverage_params
    params.require(:coverage).permit(
    attachments_attributes: [:id, :file, :_destroy],
    outcome_coverages_attributes: [:outcome_id, :_destroy]
)
```